config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-periodics.yaml

```diff
@@ -332,9 +332,5 @@ periodics:
         done;
         go work sync;
         cd -;
-        kubetest2 gce -v 2 --repo-root $REPO_ROOT \
-          --build --up --down --test=ginkgo \
-          --master-size n1-standard-2  \
-          -- \
-          --test-package-version="${TEST_PACKAGE_VERSION}" \
-          --focus-regex='\[Conformance\]'
+        kubetest2 gce -v 2 --repo-root $REPO_ROOT --build --up --down --test=ginkgo --node-size n1-standard-4 --master-size n1-standard-8 -- --test-package-version="${TEST_PACKAGE_VERSION}" --parallel=30 --focus-regex='\[Conformance\]' --skip-regex='\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]'
```

Comment:
Afaik we can't mix and match `--focus-regex` and `--skip-regex`. Has this changed?

Reply:
If that was a problem it  was only at kubetest2 level, because it always worked that way for ginkgo

I see they used in several places
https://github.com/kubernetes/test-infra/blob/490bbe3748cbbac145d24a39d1065da8951d043f/config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml#L125-L133

Reply:
LGTM for changing what we are testing.

Reply:
/lgtm